druzhba/table.py

```diff
@@ -296,6 +296,8 @@ def __init__(
         self.logger = logging.getLogger(f"druzhba.{database_alias}.{source_table_name}")
         self.s3 = Session().client("s3")
 
+        self._sql_description_cache = {}
```

Comment:
Could we call this `self._sql_description` instead? The `cache` part seems unnecessarily wordy

Reply:
I hesitated to call it `_sql_description` since it's a mapping from query to description rather than the description itself, but will update if you think that's more clear.

Reply:
`self._sql_description_mapping` then?

Reply:
Sounds good. Addressed in 722a651.